src/common/storage/backends/StorageClient.js

```diff
@@ -1,24 +1,29 @@
-/* globals define, WebGMEGlobal */
+/* globals define */
 define([
-    'client/logger'
+    'client/logger',
+    'deepforge/gmeConfig'
 ], function(
-    Logger
+    Logger,
+    gmeConfig
 ) {
+    const fetch = require.isBrowser ? window.fetch :
+        require.nodeRequire('node-fetch');
+    const Headers = require.isBrowser ? window.Headers : fetch.Headers;
     const StorageClient = function(id, name, logger) {
         this.id = id;
         this.name = name;
         if (!logger) {
-            let gmeConfig;
-            if (require.isBrowser) {
-                gmeConfig = WebGMEGlobal.gmeConfig;
-            } else {
-                gmeConfig = require.nodeRequire('../../../config');
-            }
             logger = Logger.create(`gme:storage:${id}`, gmeConfig.client.log);
         }
         this.logger = logger.fork(`storage:${id}`);
     };
 
+    StorageClient.prototype.getServerURL = function() {
```

Comment:
This should probably have a guard against when running in the browser. This currently will only work from NodeJS